cmd/README.md

```diff
@@ -20,6 +20,9 @@ Command Line tool for managing the WSO2 Micro Integrator
 
     Created Command Line tool packages will be available at cmd/build directory
 
+- #### Generating Autocompletion scripts
+    After changing commands run following to generate shell completions `go run tools/completion.go`
```

Comment:
I did not understand this. Can you explain what do you mean by "After changing commands "?

Reply:
Isn't it better if we avoid this step and just have the mi_bash_completion.sh as a resource that we can download?

Reply:
If you add new commands or change an existing command, to get the new shell completion script you must run the above command

Reply:
mi_bash_completion.sh script is added to the zip archive in 157ab87